diff --git a/bot.py b/bot.py
index c48c1d7..4bf5749 100644
--- a/bot.py
+++ b/bot.py
@@ -242,7 +242,7 @@ async def product_cycle(
 async def help(inter):
     embed = Embed(title="Bot Help")
     embed.add_field(
-        name="/get_products", value="Gets all available products", inline=False
+        name="/get_products", value="Gets all available products.", inline=False
     )
     embed.add_field(
         name="/product_details",
@@ -250,10 +250,10 @@ async def help(inter):
         inline=False,
     )
     embed.add_field(
-        name="/product_cycle", value="Gets details of a single cycle", inline=False
+        name="/product_cycle", value="Gets details of a single cycle.", inline=False
     )
     embed.add_field(
-        name="/help", value="Get help information about the bot", inline=False
+        name="/help", value="Get help information about the bot.", inline=False
     )
     embed.set_footer(text="Use /help <command> for more details on a specific command.")
     await inter.channel.send(embed=embed)
